crates/bevy_core_pipeline/src/motion_blur/motion_blur.wgsl

```diff
@@ -0,0 +1,88 @@
+// #import bevy_pbr::mesh_types
+// #import bevy_pbr::mesh_view_bindings  globals
+#import bevy_pbr::prepass_utils
+
+// #import bevy_pbr::mesh_vertex_output  MeshVertexOutput
+
+#import bevy_pbr::utils
+#import bevy_core_pipeline::fullscreen_vertex_shader FullscreenVertexOutput
+
+#ifdef MULTISAMPLED
+@group(0) @binding(0) var screen_texture: texture_2d<f32>;
+@group(0) @binding(1) var motion_vectors: texture_multisampled_2d<f32>;
+@group(0) @binding(2) var depth: texture_depth_multisampled_2d;
+#else
+@group(0) @binding(0) var screen_texture: texture_2d<f32>;
+@group(0) @binding(1) var motion_vectors: texture_2d<f32>;
+@group(0) @binding(2) var depth: texture_depth_2d;
+#endif
+@group(0) @binding(3) var texture_sampler: sampler;
+struct PostProcessSettings {
+    shutter_angle: f32,
+    max_samples: u32,
+    depth_bias: f32,
+#ifdef SIXTEEN_BYTE_ALIGNMENT
```

Comment:
Is this needed? Due to needing to sample the prepass, I think this can only run on native and webgpu.